django_components/template_loader.py

```diff
@@ -14,15 +14,9 @@ def get_dirs(self):
         component_dir = "components"
         directories = list(get_app_template_dirs(component_dir))
 
-        if settings.SETTINGS_MODULE:
-            settings_path = Path(*settings.SETTINGS_MODULE.split("."))
-
-            path = (settings_path / ".." / component_dir).resolve()
-            if path.is_dir():
-                directories.append(path)
-
-            path = (settings_path / ".." / ".." / component_dir).resolve()
-            if path.is_dir():
+        if hasattr(settings, "BASE_DIR"):
```

Comment:
BASE_DIR is not a standardized variable name that contains the base directory, so I don't think we can use it.

Reply:
Indeed